### ISSUE TITLE
New source for data AT

### ISSUE THREAD

**Issue by IanKopacka** [ec-jrc/COVID-19#24]
The data for Austria has not been updated since 2020-10-07. This was probably caused by the emigration of the official reporting dashboard in Austria.
Old link: https://info.gesundheitsministerium.at/ 
New link: https://covid19-dashboard.ages.at/

**Comment by jeffpires** [comment]
A related issue has appeared for the 11 October record for Austria-Wien, which shows a 24-hr increase of 34,675 new cases.  This does not match the country data source and far exceeds the typical <1000 daily increase.

**Comment by IanKopacka** [comment]
@jeffpires: You are right, from 2020-10-11 on the numbers for Austria-Wien for CumulativePositive, CumulativeRecovered and CurrentlyPositive seem to report the overall numbers for Austria.

The correct numbers can be found under https://covid19-dashboard.ages.at/data/data.zip

_CumulativePositive_: file "CovidFaelle_Timeline.csv", column "AnzahlFaelleSum"
_CumulativeDeceased_: file "CovidFaelle_Timeline.csv", column "AnzahlTotSum"
_CumulativeRecovered_: file "CovidFaelle_Timeline.csv", column "AnzahlGeheiltSum"
_Hospitalized_: file "CovidFallzahlen.csv", column "FZHosp"
_IntensiveCare_: file "CovidFallzahlen.csv", column "FZICU"

Would it be possible to correct the data from October 11th onwards? Many thanks!
